README.md

```diff
@@ -36,19 +36,21 @@ cuVS comes with pre-built packages that can be installed through [conda](https:/
 
 | Python | C++ | C |
 |--------|-----|---|
-| `pycuvs`| `libcuvs` | `libcuvs_c` |
+| `cuvs`| `libcuvs` | `libcuvs_c` |
```

Comment:
I don't think there is a `libcuvs_c` package, but there is a `libcuvs-static`: https://github.com/rapidsai/cuvs/blob/branch-24.08/conda/recipes/libcuvs/meta.yaml#L18

Reply:
There is a `libcuvs_c` shared library, with a C API over `libcuvs`:

https://github.com/rapidsai/cuvs/blob/9dc3a4dc5085cb5eb92d18ca1eafd5eef3dc405e/cpp/CMakeLists.txt#L556

https://github.com/rapidsai/cuvs/blob/9dc3a4dc5085cb5eb92d18ca1eafd5eef3dc405e/cpp/CMakeLists.txt#L588-L592

But you're right, it doesn't appear there's a conda package being distributed under that name.

I just pushed a commit updating the relevant docs, but maybe we should get another review from @benfred or @cjnolet to be sure the docs are representing these packages the way they'd like.

Reply:
I don't think there would be much point of a `libcuvs_c`, since it just contains the C-API that **requires** `lubcuvs`, so this change makes sense to me. Would like confirmation from @benfred that I'm not misremembering something.